api/src/pcapi/sandboxes/scripts/creators/industrial/__init__.py

```diff
@@ -2,6 +2,9 @@
     add_accessibility_compliance_to_venues,
 )
 from pcapi.sandboxes.scripts.creators.industrial.create_industrial_admin_users import *
+from pcapi.sandboxes.scripts.creators.industrial.create_industrial_app_gdpr_user_extract import (
```

Comment:
On avait des « app users », mais « app gdpr user extract » ne parle pas.
```suggestion
from pcapi.sandboxes.scripts.creators.industrial.create_industrial_gdpr_user_extracts import (
```